SpatialGDKEditorToolbar/Source/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -346,8 +346,9 @@ void FSpatialGDKEditorToolbarModule::StartSpatialOSButtonClicked()
 
 	const FString ExecuteAbsolutePath = SpatialGDKToolbarSettings->GetProjectRoot();
 	const FString CmdExecutable = TEXT("cmd.exe");
+
 	const FString SpatialCmdArgument = FString::Printf(
-		TEXT("/c spatial.exe local launch %s"), *SpatialGDKToolbarSettings->SpatialOSLaunchConfig);
+		TEXT("/c spatial.exe worker build build-config & spatial.exe local launch %s"), *SpatialGDKToolbarSettings->SpatialOSLaunchConfig);
```

Comment:
Is this causing additional noticeable delay?

Reply:
Yes, a few seconds but not more, it will launch two command prompt windows after each other though